src/Microsoft.DotNet.Watcher.Tools/project.json

```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Command line tool to watch for source files changes during development and restart dotnet.",
```

Comment:
How about:
* `... source file changes...` (singular `file`).
 *`... and restart the dotnet command.`